core/modules/user.py

```diff
@@ -233,7 +233,7 @@ def pwrecover(self, authtoken=None, skey=None, *args, **kwargs):
 				60 * 60 * 24,
 				userKey=utils.normalizeKey(user.key),
 				password=skel["password"])
-			utils.sendEMail([userSkel["name"]], self.userModule.passwordRecoveryMail, userSkel)
+			email.sendEMail([userSkel["name"]], self.userModule.passwordRecoveryMail, userSkel)
```

Comment:
The parameters are keyword-only, so your fix is still incomplete.
```suggestion
			email.sendEMail(dests=userSkel["name"], tpl=self.userModule.passwordRecoveryMail, skel=userSkel)
```